client/src/site-search/search-results.tsx

```diff
@@ -428,21 +470,25 @@ function Pagination({
       }
     }
     return (
-      <div className="pagination">
+      <ul className="pagination">
         {previousURL ? (
-          <Link to={previousURL} className="button">
-            Previous
-          </Link>
+          <li>
+            <Link to={previousURL} className="button">
+              Previous
+            </Link>
+          </li>
         ) : null}{" "}
         {nextPage ? (
-          <Link
-            to={`?${appendURL(searchParams, { page: `${nextPage}` })}`}
-            className="button"
-          >
-            Next
-          </Link>
+          <li>
+            <Link
+              to={`?${appendURL(searchParams, { page: `${nextPage}` })}`}
+              className="button"
+            >
+              Next
+            </Link>
+          </li>
         ) : null}
-      </div>
+      </ul>
```

Comment:
Would it be possible to make this consistent with the Prev/Next buttons we use in the content? I.e. move the "Next" to the right. 
<img width="955" alt="Screen Shot 2021-02-23 at 9 14 46 AM" src="https://user-images.githubusercontent.com/26739/108856144-bfaa3500-75b7-11eb-92bb-94fac89b2500.png">

Reply:
Done.